src/styles/main.scss

```diff
@@ -0,0 +1,78 @@
+$clock-hand-width: 20px;
+
+@keyframes move-watch-hand {
+  0% {
+    transform: rotate(-180deg);
+    transform-origin: ($clock-hand-width / 2) ($clock-hand-width / 2);
+  }
+
+  25% {
+    transform: rotate(-90deg);
+    transform-origin: ($clock-hand-width / 2) ($clock-hand-width / 2);
+  }
+  
+  50% {
+    transform: rotate(0deg);
+    transform-origin: ($clock-hand-width / 2) ($clock-hand-width / 2);
+  }
+
+  75% {
+    transform: rotate(90deg);
+    transform-origin: ($clock-hand-width / 2) ($clock-hand-width / 2);
+  }
```

Comment:
I think these interim state could be removed, 0% and 100% is pretty enough for animation to work